lib/score.js

```diff
@@ -0,0 +1,63 @@
+Score = {
+	send_team_high_scores: function(){
+		Logger.debug('Score.send_team_high_scores');
+		var team_scores = {};
+
+		for (var i = Game.players.length - 1; i >= 0; i--) {
+			var player = Game.players[i];
+			var team = player.team;
+
+			if (!team){
+				continue;
+			}
+
+			var score = player.score.seconds;
+			var speed = player.score.speed;
+
+			if (team_scores.hasOwnProperty(team)){
```

Comment:
This function and the one below for DB query call back look a lot alike. Why not create a helper function that would add a new team score found to existing array or team scores.